### ISSUE TITLE
Multiple jax-rs instrumentations

### ISSUE THREAD

**Issue by pavolloffay** [opentracing-contrib/meta#15]
There are multiple jax-rs instrumentation:

1. https://github.com/opentracing-contrib/java-jaxrs
2. https://github.com/uber/jaeger-client-java/tree/master/jaeger-jaxrs2 @yurishkuro 
3. https://github.com/opentracing-contrib/java-dropwizard/blob/master/dropwizard-opentracing/src/main/java/io/opentracing/contrib/dropwizard/ServerRequestTracingFilter.java @bhs 

https://github.com/opentracing-contrib/java-jaxrs ropadmap:
* use with combination with https://github.com/opentracing-contrib/java-web-servlet-filter to capture errors like no URL mapping, unauthenticated requests, exceptions in filters...
* in-process propagation, as intermediate solution maybe use `spanmanager` , once https://github.com/opentracing/specification/issues/23 is solved use what standard says.

Would it make sense for you to reuse `java-jaxrs` in 2. and 3.?

**Comment by yurishkuro** [comment]
> Would it make sense for you to reuse java-jaxrs in 2. and 3.?

Yes, but I would wait for things with span manager / global tracer to settle.
